website/docusaurus.config.js

```diff
@@ -3,18 +3,18 @@ module.exports = {
   title: 'Next.js API Decorators',
   tagline:
     'Collection of decorators to create typed Next.js API routes, with easy request validation and transformation.',
-  url: 'https://storyblok-toolkit.vercel.com',
+  url: 'https://next-api-decorators.vercel.com',
   baseUrl: '/',
   onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'warn',
   favicon: 'img/favicon.ico',
-  organizationName: 'storyofams',
+  organizationName: 'instantcommerce',
```

Comment:
Instant Commerce? @leeuwis is there a prefered way of writing this?

Reply:
it's the github organization name, so should stay as is.

https://docusaurus.io/docs/api/docusaurus-config#organizationName